sorts/bubble_sort.py

```diff
@@ -45,8 +45,7 @@ def bubble_sort(collection):
 
     doctest.testmod()
 
-    user_input = input("Enter numbers separated by a comma:").strip()
-    unsorted = [int(item) for item in user_input.split(",")]
+    unsorted = list(map(int,input("Enter numbers separated by Space : ").split()))
```

Comment:
CONTRIBUTING.md recommends not using map() for readability reasons.  Also see "What's new in Python 3".

Reply:
okay understood.